html-template/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="theme-color" content="#000000">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
please can you explain what this change is for

Reply:
This code was added so the browser could relate the screen size to the device being used, eg Desktop, Samsung S5 etc.
Manually resizing the browser window will render the correct font but not update the value in dev tools. This meta tag ensures that dev-tools output shows the current font size.